CONTRIBUTING.md

```diff
@@ -9,10 +9,16 @@ Run `terminus self:info` to confirm you are [running the latest version](https:/
 Setting Up
 ----------
 
-1. Clone this Git repository on your local machine.
-2. Install [Composer](https://getcomposer.org/) if you don't already have it.
-3. Run `composer install` to fetch all the dependencies.
-4. Run `./bin/terminus --help` to test that everything was installed properly.
+Fork this repo then open a terminal session and execute the following:
```

Comment:
This stuff seems unrelated. Is this an artifact from the base branch?

Reply:
Thank you for the feedback @jazzsequence.  Removed!